Sources/NIO/ByteBuffer-views.swift

```diff
@@ -66,6 +66,13 @@ public struct ByteBufferView: RandomAccessCollection {
             return try body(bytes.bindMemory(to: UInt8.self))
         }
     }
+
+    /// Returns a `ByteBuffer` slice from the same range as the view.
+    ///
+    /// - returns: A `ByteBuffer` slice from the same range as the view.
+    public func getByteBuffer() -> ByteBuffer {
```

Comment:
I don't think the naming is consistent with Swift(NIO). I could see a couple of options:

1. `public var byteBuffer: ByteBuffer { ... }`, fine as a property because it's constant time
2. `public var buffer: ByteBuffer { ... }`
3. `public func as(Byte)Buffer() -> ByteBuffer`
4.  as a `ByteBuffer` initialiser

```
extension ByteBuffer {
    public init(_ view: ByteBufferView) {
        self = view.buffer.getSlice(....)
    }
}
```

CC @Lukasa ?

Reply:
You're right, I forgot that `readableBytesView` is non-optional. 
In that case, I think the initializer would be the most natural choice. Especially if you compare it with `Array` and `ArraySlice`, where you use the `Array` initializer to convert the slice back into an array.

Reply:
thanks! sounds good to me. @Lukasa happy with this too?

Reply:
Works for me!

Reply:
Perfect 👌 I'll change the naming & implement the unit test suggestions from @weissi.